source/configure/authentication-configuration-settings.rst

```diff
@@ -2036,69 +2036,72 @@ Office 365 OAuth 2.0 settings
+| - **false**: **(Default)** Disables Office 365 OAuth authentication.                                                                                                         | - ``config.json`` setting: ``Office365Settings.Enable: false`` |
+|                                                                                                                                                                              | - Environment variable: ``MM_OFFICE365SETTINGS_ENABLE``        |
+|                                                                                                                                                                              |                                                                |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------+
 
-**False**: Office 365 OAuth cannot be used for team creation or account signup.
-
-+----------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"Enable": false`` with options ``true`` and ``false``. |
-+----------------------------------------------------------------------------------------------------+
+Office 365 OAuth 2.0 Application ID
+'''''''''''''''''''''''''''''''''''
 
 .. config:setting:: oauth-o365appid
   :displayname: Application ID (OAuth - Office 365)
   :systemconsole: Authentication > OAuth 2.0
   :configjson: Id
   :environment: N/A
-  :description: Obtain this value by registering Mattermost as an application in your Microsoft or Office account.
-
-Application ID
-''''''''''''''
+  :description: This setting holds the Application ID generated when configuring Office 365 as a Single Sign-On service through the Microsoft Azure Portal.
 
 *Available in legacy Enterprise Edition E20*
 
-Obtain this value by registering Mattermost as an application in your Microsoft or Office account.
++-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------+
+| This setting holds the **Application ID** generated when configuring Office 365 as a Single Sign-On service through the Microsoft Azure Portal. See `Register an application in Azure Portal <https://docs.mattermost.com/onboard/sso-office.html#step-1-register-an-application-in-azure-portal>`__. | - System Config path: **Authentication > OAuth 2.0** |
```

Comment:
See my comment on https://github.com/mattermost/docs/pull/6172#discussion_r1085382223 about the format used to provide docs references and the feedback via https://github.com/mattermost/docs/pull/6132#issuecomment-1402051426 about potentially moving all docs references to a subsequent row for consistency.